controller/src/main/java/io/pravega/controller/store/stream/Stream.java

```diff
@@ -363,6 +363,13 @@
      */
     CompletableFuture<Pair<Integer, List<Integer>>> getActiveEpoch(boolean ignoreCached);
 
+    /**
+     * Method to get stream size till the given stream cut
+     * @param streamCut stream cut
+     * @return size of stream till given cut
```

Comment:
This Javadoc does not correspond to the guidelines (we will enable in a few days). Please be explicit about what this returns (a `CompletableFuture`). See https://github.com/pravega/pravega/wiki/Contributing#javadoc-guidelines

Reply:
done